spec/acceptance/steps/steps.rb

```diff
@@ -1,32 +1,54 @@
-step 'there is a home page' do
+CORRECT_EMAIL = 'lopandya96@gmail.com'
+CORRECT_PASSWORD = 'lopandya96'
+INCORRECT_EMAIL = 'FALSE@i.ua'
+INCORRECT_PASSWORD = 'FALSE'
+
+step 'I am on home page' do
   visit '/'
 end
 
-step 'there is a login page' do
+step 'I am on login page' do
   visit '/users/sign_in'
 end
 
 step 'I click login' do
   click_link('Login')
 end
 
-step 'login page should open' do
+step 'I should see login page' do
   expect(page).to have_current_path('/users/sign_in')
 end
 
-step 'I fill :email in email and :password in password' do |email, password|
+step "I fill out :email email and :password password" do |email, password|
```

Comment:
Do not mix several steps into single one. It introduces extra complexity in step definitions which should be ideally 3 lines of code